docs/operation/cluster_and_service_configuration.rst

```diff
@@ -0,0 +1,206 @@
+Cluster and Services Configurations
+===================================
+
+This section contains information describing the list of available Cluster and
+Services Configurations including procedures for customizing and applying any
+given Cluster and Services Configurations.
+
+Managing Cluster and Services Configurations
+********************************************
+
+Newly deployed **MetalK8s** clusters come with chosen default values for most
+Cluster services. These default values are transparent to Admin users
+and can be customized at any point in time given that Admins follow the
+documented procedure to the later.
+
+Managing Authentication
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Add a local static user
+"""""""""""""""""""""""
+
+Local authentication via static users is enabled by default after a fresh
+MetalK8s installation.
+
+   .. important::
+
+      To continue using MetalK8s in cases where the external authentication
+      system fails, we advise MetalK8s administrators to leave the default
+      super admin account enabled at all times.
+
+To add a new static user, perform the following operations:
+
+#. Generate a password hash. The command below requires **apache-utils**
+   package to be installed.
+
+   .. code-block:: shell
+
+      $ htpasswd -bnBC 12 "" <replace-with-password> | tr -d ':\n'
+
+#. From the Bootstrap node, edit the ConfigMap ``metalk8s-dex-config`` and then
+   add a new entry to the already existing list using:
+
+   .. code-block:: shell
+
+      $ kubectl --kubeconfig /etc/kubernetes/admin.conf \
+         edit configmaps metalk8s-dex-config -n metalk8s-auth
+
+   The new entry should be unique and possess mandatory fields like ``email``,
+   ``hash``, ``username`` and ``userID`` like in the example below.
+
+   .. code-block:: yaml
+
+      data:
+         config.yaml: |-
+            spec:
+               localuserstore:
+                  userlist:
+                    - email: "<email>"
+                      hash: "<replace-with-hash>"
+                      username: "<username>"
+                      userID: "<uuidv4>"
+
+#. Save and apply the changes.
+
+
+#. Connect to the Salt master container using:
+
+   .. code-block:: shell
+
+      $ crictl exec -it $(crictl ps -q --label io.kubernetes.container.name=salt-master) bash
+
+#. From the Salt master container, apply the following salt-states to propagate
+   the new changes down to the underlying services.
+
+   .. code-block:: shell
+
+      $ salt-run state.sls metalk8s.addons.dex.deployed saltenv=metalk8s-<version>
+
+#. Finally, create and apply the required :file:`ClusterRoleBinding.yaml` file
+   that ensures that the newly added static user is bound to a Cluster Role.
+
+   .. note::
+
+      To obtain the list of available Cluster Roles in a MetalK8s cluster,
+      use the following command:
+
+      .. code-block:: shell
+
+         kubectl --kubeconfig /etc/kubernetes/admin.conf get clusterroles
```

Comment:
when we execute this command, we have around 70 cluster roles. What cluster role should we choose?

Reply:
I have updated this section!

Reply:
yes I like it